modules/nw-configuring-qinq-sriov-proc.adoc

```diff
@@ -0,0 +1,199 @@
++
+[source,terminal]
+----
+$ watch -n 1 'oc get sriovnetworknodestates -n openshift-sriov-network-operator <node_name> -o jsonpath="{.status.syncStatus}"'
+----
++
+The synchronization status indicates a change from `InProgress` to `Succeeded`.
+
+. Create a `SriovNetwork` object, and set the outer VLAN called the S-tag, or `Service Tag`, as it belongs to the infrastructure.
++
+[IMPORTANT]
+====
+You must configure the VLAN on the trunk interface of the switch. In addition, you might need to further configure some switches to support QinQ tagging.
+====
+
+.. Create a file named `nad-sriovnetwork-1ad-810.yaml` by using the following content:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriovnetwork-1ad-810
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: '{}'
+  vlan: 171 <1>
+  vlanProto: "802.1ad" <2>
+  networkNamespace: default
+  resourceName: resource810
+----
++
+<1> Sets the S-tag VLAN tag to `171`. 
+<2> Specifies the VLAN protocol to assign to the virtual function (VF). Supported values are `802.1ad` and `802.1q`. The default value is `802.1q`. 
+
+.. To create the object, enter the following command:
++
+[source,terminal]
+----
+$ oc create -f nad-sriovnetwork-1ad-810.yaml
+----
+
+. Create a Network Attachment Definition with an inner VLAN. The inner VLAN is often referred to as the C-tag, or `Customer Tag`, as it belongs to the Network Function:
+
+.. Create a file named `nad-cvlan100.yaml` by using the following content:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: nad-cvlan100
+  namespace: default
+spec:
+  config: '{
+    "name": "vlan-100",
+    "cniVersion": "0.3.1",
+    "type": "vlan",
+    "linkInContainer": true,
+    "master": "net1", <1>
+    "vlanId": 100,
+    "ipam": {"type": "static"}
+  }'
+----
++
+<1> Specifies the VF interface inside the pod. The default name is `net1` as the name is not set in the pod annotation. 
+
+.. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f nad-cvlan100.yaml
+----
+
+.Verification
+
+- Verify QinQ is active on the node by following this procedure: 
+
+. Create a file named `test-qinq-pod.yaml`  by using the following content:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: sriovnetwork-1ad-810, nad-cvlan100
+spec:
+  containers:
+    - name: test-container
+      image: quay.io/ocp-edge-qe/cnf-gotests-client:v4.10
+      imagePullPolicy: Always
+      securityContext:
+        privileged: true
+----
+
+. Create the test pod by running the following command:
++
+[source,terminal]
+----
+$ oc create -f test-qinq-pod.yaml
+---- 
+
+. Obtain a list of cluster nodes by running the following command: 
```

Comment:
please use the same convention as in https://docs.openshift.com/container-platform/4.15/support/gathering-cluster-data.html#support-collecting-network-trace_gathering-cluster-data

Also please consider to use something more direct like

oc debug node/cnfdr28.telco5g.eng.rdu2.redhat.com -- bash -c "ip link show eno12409"

Its more concise, you can say that on the node where the pod is present, run the following command and thatś it